client/cherami/outputhostconnection.go

```diff
@@ -161,7 +159,7 @@ func (conn *outputHostConnection) close() {
 		}
```

Comment:
samarabbas, isOpened() and isClosed() access the member without holding the lock. So, leaving as is.